packages/benchmarks/src/textToDriver/evaluationMetrics.ts

```diff
@@ -0,0 +1,41 @@
+import { TextToDriverEvalScorer } from "./evalTypes";
+
+/**
+  Check if the generated query successfully executed.
+  Note that if the query isn't valid, then the "SuccessfulExecution"
+  metric will always fail.
+ */
+export const SuccessfulExecution: TextToDriverEvalScorer = async ({
+  output,
+}) => {
+  const noOutput = output.execution.result === null;
+
+  return {
+    name: "SuccessfulExecution",
+    score: noOutput ? 0 : 1,
+    ...(output.execution.error
+      ? {
+          metadata: {
+            error: output.execution.error,
+          },
+        }
+      : {}),
+  };
+};
+
+/**
+  Measure how long the query takes to execute in minutes.
+
+  Note: Measuring in minutes because
+  Braintrust throws an error if the score > 1.
```

Comment:
Not sure I totally follow this comment. Is the idea that we expect all execution times to be under 1 minute? Might be true now but not in the future, e.g. for large aggregations

Reply:
yea exactly. this is limited in that respect